cmd/root.go

```diff
@@ -99,11 +99,11 @@ Welcome to the Astro CLI, the modern command line interface for data orchestrati
 		newRunCommand(),
 	)
 
-	if config.CFG.SQLCLI.GetBool() {
-		rootCmd.AddCommand(
-			sql.NewFlowCommand(),
-		)
-	}
+	// if config.CFG.SQLCLI.GetBool() {
```

Comment:
@dimberman why did we remove this..?

I think this line is still necessary, at least while the SQL/flow commands are alpha. We only want to show the flow commands to end-users if they do:
```
astro config set -g beta.sql_cli true
```